ml/xgboost/src/main/java/ai/djl/ml/xgboost/XgbModel.java

```diff
@@ -54,6 +55,20 @@ public void load(Path modelPath, String prefix, Map<String, ?> options) throws I
             }
         }
         block = JniUtils.loadModel((XgbNDManager) manager, modelFile.toAbsolutePath().toString());
+        // set extra options
+        if (options != null) {
+            if (options.containsKey("Mode")) {
+                ((XgbSymbolBlock) block)
+                        .setMode(
+                                XgbSymbolBlock.Mode.valueOf(
+                                        ((String) options.get("Mode"))
+                                                .toUpperCase(Locale.ENGLISH)));
```

Comment:
1. Not sure if toUpperCase is needed. I prefer let user set it correctly.
2. User Locale.ROOT instead